### PR TITLE
Products filter Screen conversion from static to react

### PR DESCRIPTION
**Pull request by Naveenchandar**
Products filter screen conversion of static to react and api integration changes

### CODE REVIEW

**Review by maitrakhatri — APPROVED** [review]
Clean code, easy to read
Liked that you have destructured props rather than using `props.something`
Suggestion: merge the basic react/mockbee setup in another branch and merge it, as it has nothing to get reviewed. It will become easy to review the main features code

File: src/pages/Products/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import './index.css';
+import ProductCard from '../../components/ProductCard';
+import { Oval } from 'react-loader-spinner';
+import ProductFilter from '../../components/ProductFilter';
+import { useProducts } from '../../context/products';
+import { LoadProducts } from './loadProducts';
+import { useDocumentTitle } from '../../hooks/useDocumentTitle';
```
Comment (rohanmathur91) [inline src/pages/Products/index.js]: try barrel exports

**Review by rohanmathur91 — APPROVED** [review]
- Good breakdown of components
 - Nice use of custom hook for changing the title
 - File structure also looks goos

File: src/pages/Products/index.js (hunk 1 above)
Comment (Naveenchandar) [reply src/pages/Products/index.js]: Sure.

**Pull request merged**